docs/overview/excel.md

```diff
@@ -68,7 +68,15 @@ More details about sharing and unsharing scripts can be in the article [Sharing
 > [!NOTE]
 > Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
 
-## Connecting Office Scripts to Power Automate
+### Create script buttons (preview)
+
+[!INCLUDE [Public preview](../includes/preview-note.md)]
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook. With any script, go to the **More options (…)** menu and select **Add button**. This creates a button in the workbook that runs the associated script when pressed. It also shares the script with the workbook, so everyone with access can use your helpful automation.
+
```

Comment:
It's true that the 'add button' option is in the code editor's menu, but I think the screenshot shows the Script Detail pane's menu